SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCore/Operations/DataBrokerOperation.swift

```diff
@@ -196,21 +247,22 @@ public class DataBrokerOperation: Operation, @unchecked Sendable {
             do {
                 Logger.dataBrokerProtection.log("Running operation: \(String(describing: operationData), privacy: .public)")
 
-                try await DataBrokerProfileQueryOperationManager(vpnBypassService: operationDependencies.vpnBypassService).runOperation(operationData: operationData,
-                                                                                brokerProfileQueryData: brokerProfileData,
-                                                                                database: operationDependencies.database,
-                                                                                notificationCenter: operationDependencies.notificationCenter,
-                                                                                runner: operationDependencies.runner,
-                                                                                pixelHandler: operationDependencies.pixelHandler,
-                                                                                showWebView: showWebView,
-                                                                                isImmediateOperation: operationType == .manualScan,
-                                                                                                                                                                                     eventsHandler: operationDependencies.eventsHandler,
-                                                                                shouldRunNextStep: { [weak self] in
-                    guard let self = self else { return false }
-                    return !self.isCancelled
-                })
-
-                let sleepInterval = operationDependencies.config.intervalBetweenSameBrokerOperations
+                try await DataBrokerProfileQueryOperationManager(vpnBypassService: operationDependencies.vpnBypassService).runOperation(
+                    dependencies: operationDependencies,
+                    operationData: operationData,
+                    brokerProfileQueryData: brokerProfileData,
+                    database: operationDependencies.database,
```

Comment:
~I think it would be better to not pass these in individually since it's also being passed the full operationDependencies anyway, it makes the function signature look quite confused~ (although forgive me if you've done this in the next PR I haven't looked at yet 😅 )
haha yep, just got that far

Reply:
Haha yes sorry, I debated just doing it in this PR as well tbh... not entirely sure why I didn't?